### PR TITLE
[Validator] Add missing translations for german and vietnamese

### PR DESCRIPTION
**Pull request by jschaedl**
| Q             | A
| ------------- | ---
| Branch?       | 3.4 
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| License       | MIT

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.vi.xlf [hunk 1]
```diff
@@ -382,6 +382,10 @@
                 <source>Each element of this collection should satisfy its own set of constraints.</source>
                 <target>Mỗi phần tử trong bộ sưu tập này nên thỏa mãn những ràng buộc của nó.</target>
             </trans-unit>
+            <trans-unit id="99">
+                <source>This value is not a valid International Securities Identification Number (ISIN).</source>
+                <target>Giá trị này không phải là International Securities Identification Number (ISIN).</target>
```
Comment (jschaedl) [inline src/Symfony/Component/Validator/Resources/translations/validators.vi.xlf:387]: @van2412 Can you please have a look? I am not sure how we should translate "International Securities Identification Number". Thank you :-)

Comment (jschaedl) [inline src/Symfony/Component/Validator/Resources/translations/validators.vi.xlf:387]:
```suggestion
                <target>Giá trị này không phải là International Securities Identification Number (ISIN) hợp lệ.</target>
```

Comment (seriquynh) [reply src/Symfony/Component/Validator/Resources/translations/validators.vi.xlf]: I'm a Vietnamese person. I think it should be "Giá trị này không phải là Mã số chứng khoán quốc tế (ISIN).". Anyway, let's wait for your friend's response.

Comment (van2412) [reply src/Symfony/Component/Validator/Resources/translations/validators.vi.xlf]: @seriquynh This looks good to me. But I suggest adding the translation of "valid" as well. Like `Giá trị này không phải là Mã số chứng khoán quốc tế (ISIN) hợp lệ`

File: src/Symfony/Component/Validator/Resources/translations/validators.vi.xlf [hunk 2]
```diff
@@ -382,6 +382,10 @@
                 <source>Each element of this collection should satisfy its own set of constraints.</source>
                 <target>Mỗi phần tử trong bộ sưu tập này nên thỏa mãn những ràng buộc của nó.</target>
             </trans-unit>
+            <trans-unit id="99">
+                <source>This value is not a valid International Securities Identification Number (ISIN).</source>
+                <target>Giá trị này không phải là International Securities Identification Number (ISIN) hợp lệ.</target>
```
Comment (van2412) [inline src/Symfony/Component/Validator/Resources/translations/validators.vi.xlf:387]:
```suggestion
                <target>Giá trị này không phải là mã số chứng khoán quốc tế (ISIN) hợp lệ.</target>
```

File: src/Symfony/Component/Validator/Resources/translations/validators.vi.xlf (hunk 1 above)
Comment (seriquynh) [reply src/Symfony/Component/Validator/Resources/translations/validators.vi.xlf]: I agree with your translation message.

Comment (jschaedl) [reply src/Symfony/Component/Validator/Resources/translations/validators.vi.xlf]: Thank you @seriquynh and @van2412

**Review by van2412 — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @jschaedl.

**Pull request closed without merging**